app/views/layouts/mailer.html.erb

```diff
@@ -3,11 +3,188 @@
+        text-align: center;
+      }
+      .main .button {
+        display: inline-block;
+        background-color: #378137; /* #298DCC in pro */
+        border-radius: 5px;
+        padding: 8px 15px;
+        color: #fff;
+        margin: 1rem 0 0.5rem 0;
+      }
+      .main .button:hover {
+        background-color: #204b20; /* #2070a2 in pro */
+      }
+      .main .digest-header {
+        font-weight: 300;
+        margin: 1rem 0 1.5rem 0;
+      }
+      .main .project-wrapper {
+        display: flex;
+        flex-direction: column;
+      }
+      .main hr {
+        width: 100%;
+        height: 0;
+        border-top: 0.5px solid #ddd;
+        border-bottom: 0.5px solid #ddd;
+        border-right: 0.5px solid transparent;
+        border-left: 0.5px solid transparent;
+      }
+      .main .project-wrapper .notification {
+        display: flex;
+        align-items: center;
+        margin: 1rem 0;
+      }
+      .main .project-wrapper .notification .avatar {
+        height: 50px;
+        width: 50px;
+        border-radius: 50%;
+        border: 1px solid #ddd;
+        margin: 0 20px 0 10px;
+      }
+      .main .project-wrapper .notification .avatar img {
+        height: 50px;
+        width: 50px;
+        object-fit: cover;
+        border-radius: 50%;
+      }
+      .main .project-wrapper .notification .notification-text {
+        text-align: left;
+        margin: 0;
+        font-weight: 400;
+      }
+      .main .project-wrapper .notification .notification-text .user-name {
+        font-weight: 600;
+      }
+      .main .project-wrapper .notification .notification-text .time-stamp {
+        font-size: 80%;
+        color: #777;
+        white-space: nowrap;
+      }
+      @media (max-width: 320px) {
+        .main .project-wrapper .notification .notification-text .time-stamp {
+          display: block;
+        }
+      }
+      .main .project-wrapper .project-name-wrapper {
+        display: flex;
+        flex-direction: row;
+        white-space: nowrap;
+        align-items: center;
+      }
+      .main .project-wrapper .project-name-wrapper .project-name {
+        font-weight: 700;
+        font-size: 1.1rem;
+        margin: 0 0.5rem;
+        line-height: 1.2rem;
+      }
     </style>
   </head>
 
   <body>
-    <%= yield %>
+    <div class="mail-wrapper">
+
+      <div class="header"> <!-- add class "center" if there is no gravatar or profile picture available for this user -->
+        <div class="logo">
+          <%= image_tag attachments['DradisCE_full_small'].url %>
+        </div>
+        <div class="gravatar"> <!-- Gravatar of the user recieving this email. Only render this div if a gravatar image is available or if the user uploaded a custom profile pic in pro. Do not render this div if the fallback/placeholder image will be rendered. If the is no gravatar that will be rendered, add class "center" to div.header above. -->
+          <%= image_tag @avatar_url, alt: @user.email %>
```

Comment:
`avatar_url` is always set to the attached profile image and won't actually attempt to load the gravatar.